### ISSUE TITLE
LTPA SSO failure for certain usernames

### ISSUE THREAD

**Issue by Zech-Hein** [OpenLiberty/open-liberty#21837]
**Describe the bug**  
Certain users may be able to login but may fail to use their LTPAToken2 in subsequent requests to other servers or to the same server if authentication Cache is disabled.

This will affect users with at least one of the following in their username: 
_space character, tab character, newline character, carriage-return character, and form-feed character._

**Steps to Reproduce**  
1.) Create a user with one of the characters above( most common will be a whitespace)
`<user name="alice space"  password="test1" />`

2.) Disable the authentication Cache (or make the step 4 request to different server)
`<authentication id="Basic" cacheEnabled="false" />`

3.) Login as the user to receive an LTPAToken2

4.) Finally, use the LTPAToken2 in another request and it will fail verification.

**Expected behavior**  
the LTPAToken2 should be verified successfully in subsequent requests.

**Diagnostic information:**  
 - OpenLiberty Version: [e.g. 22.0.0.8 ]
 - Affected feature(s) [e.g. appSecurity-2.0, appSecurity-3.0]
 - Java Version: [any java version]
 - server.xml configuration  
```
<user name="alice space"  password="test1" />
<authentication id="Basic" cacheEnabled="false" />
```
 - messages found in `$WLP_OUTPUT_DIR/messages.log`
 ```
 [7/21/22 19:18:51:127 EDT] 000000fa id=00000000 com.ibm.ws.security.token.internal.TokenManagerImpl          I CWWKS4001I: The security token cannot be validated. This can be for the following reasons
1. The security token was generated on another server using different keys.
2. The token configuration or the security keys of the token service which created the token has been changed.
3. The token service which created the token is no longer available.
```

**Comment by barbj** [comment]
Fixed by #21835

**Comment by barbj** [comment]
Putting release bug back on since, although the code this fixed hadn't been shipped in a fixpack, it did ship in an ifix.
